release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -68,6 +68,12 @@ With this release, you can now install {op-system} to iSCSI boot devices. Multip
 [id="ocp-4-16-installation-and-update_{context}"]
 === Installation and update
 
+Ingress capability is now a configurable cluster capability. It must be enabled for OpenShift clusters to start.
```

Comment:
The description looks confused here, do we need to mention that it is optional capability for hypershift, but always enabled in standalone OCP? 
@alebedev87 @lihongan  what's your opinion for this?

Reply:
I think ingress capability is not configurable in standalone OCP.
cc @melvinjoseph86

Reply:
yes, we need to mention this as option capability to Hypershift and always enabled for standalone OCP